### ISSUE TITLE
Browser version - no button to close the sidecar?

### ISSUE THREAD

**Issue by showeimer** [IBM/kui#1691]
There doesn't seem to be a button to close out/collapse the sidecar.  The user may want to close the sidecar so that they get back the full width of the main view, the are some responsiveness issues with certain tables where data is getting truncated and being unable to close the sidecar increases the problem.

**Comment by myan9** [comment]
Thanks! We do have button to close the sidecar. Have you tried the button below? It's located at the right corner. You can also use ESC to close the sidecar. 
![Screen Shot 2019-06-13 at 10 51 37 AM](https://user-images.githubusercontent.com/21212160/59442991-3d6ce380-8dc9-11e9-983e-0b5670f65148.png)

**Comment by showeimer** [comment]
@myan9 interesting!  I did not see/think that would button would do that.  It does appear that button is more of a collapse feature, as opposed to a full close out button.  I did notice that after I collapsed and re-expanded the sidecar, the YAML text gets truncated:

<img width="305" alt="Screen Shot 2019-06-13 at 11 24 27 AM" src="https://user-images.githubusercontent.com/26075187/59445533-d7368f80-8dcd-11e9-8fc2-21ab573a8f9c.png">
